static/js/components/AssignmentList.jsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import * as Actions from '../ducks/source';
+import * as UserActions from '../ducks/users';
+import styles from './AssignmentList.css';
```

Comment:
I see you use both `makeStyles` and `css` in this PR; `makeStyles` is preferred.  However, since we have a mish-mash at the moment, you can leave this as is if you prefer.

Reply:
Merged with `makeStyles`.

Reply:
Actually, leaving as is for now -- will file an issue for moving everything from css to css-in-jsx.